<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSmallScreen</class>
 <widget class="QFrame" name="CSmallScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="cameraFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#cameraFrame
{
	background:rgb(83,83,83);
}</string>
     </property>
     <widget class="BetaPlayer" name="camera_left">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>193</width>
        <height>105</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color:white;
background:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>请稍后...</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="BetaPlayer" name="camera_right">
      <property name="geometry">
       <rect>
        <x>193</x>
        <y>0</y>
        <width>193</width>
        <height>105</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
background:#535353;
border:none;
}</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>请稍后...</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{

font-size: 16px;
font-family: PingFang-SC-Bold, PingFang-SC;
font-weight: bold;
color: rgb(116,116,116);
line-height: 22px;
}

.QLabel[screen24=&quot;true&quot;] 
{
	font-size: 16px;
	color: rgb(116,116,116);
}

.QLabel[screen12=&quot;true&quot;] 
{
	font-size: 24px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="act_focus">
   <property name="text">
    <string>大屏查看</string>
   </property>
   <property name="toolTip">
    <string>大屏查看</string>
   </property>
  </action>
  <action name="act_swap_seat">
   <property name="text">
    <string>交换座位</string>
   </property>
   <property name="toolTip">
    <string>交换座位</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>打开视频</string>
   </property>
   <property name="toolTip">
    <string>打开视频</string>
   </property>
  </action>
  <action name="act_close">
   <property name="text">
    <string>关闭视频</string>
   </property>
   <property name="toolTip">
    <string>关闭视频</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BetaPlayer</class>
   <extends>QLabel</extends>
   <header>BetaPlayer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
